libavcodec/hevcdec.c

```diff
@@ -324,7 +324,7 @@ static void export_stream_params(HEVCContext *s, const HEVCSPS *sps)
 {
     AVCodecContext *avctx = s->avctx;
     const HEVCParamSets *ps = &s->ps;
-    const HEVCVPS *vps = ps->vps_list[sps->vps_id];
+    HEVCVPS *vps = NULL;
```

Comment:
don't remove the const.
const STRUCT * means the pointer cannot be used to change the struct.
STRUCT * const means the pointer variable cannot be changed

Reply:
Done.